### DISCUSSION TITLE
How to enable Remote Debug Remote Debug? (visualize flow of StateMachine & inspect parameters in realtime)?

### DISCUSSION THREAD

**Discussion by Krasapan** [imjp94/gd-YAFSM#38] [Q&A] [answered]
I saw that you have that great feature, but for some reason when I run the game it doesn't visualize any parameters changes. How to turn it on?

**Comment by imjp94** [comment, marked as answer]
To view remote debugger, you have to select the `StateMachinePlayer` node from Remote Scene Tree when playing the game. [See it in action from this video](https://twitter.com/imjp94/status/1383744743369179147)
